bootstrap.d/dev-libs.yml

```diff
@@ -526,6 +526,40 @@ packages:
           DESTDIR: '@THIS_COLLECT_DIR@'
         quiet: true
 
+  - name: libotf
+    source:
+      git: 'https://salsa.debian.org/debian/libotf.git'
```

Comment:
This does not seem to be the upstream repo (?) but I also could not find the upstream one.

Reply:
Where do other distros like Void or Gentoo build from?

Reply:
Also, I'm missing `subdir: 'ports'` here.

Reply:
> Where do other distros like Void or Gentoo build from?

Gentoo builds from the official website of the project. But that is only a tarball. So i opted for a github mirror as i heard that it is preferred.

Reply:
> Also, I'm missing `subdir: 'ports'` here.

Fixed it

Reply:
> 
> 
> > Where do other distros like Void or Gentoo build from?
> 
> 
> 
> Gentoo builds from the official website of the project. But that is only a tarball. So i opted for a github mirror as i heard that it is preferred.
> 
> 

Regarding Void, they also build from a tarball. As this seems to be the Debian mirror, I’d say that is good enough.